pwnlib/context/__init__.py

```diff
@@ -972,7 +972,10 @@ def log_level(self, value):
         except AttributeError:  pass
 
         # Otherwise, fail
-        level_names = filter(lambda x: isinstance(x,str), logging._levelNames)
+        if sys.version_info[0] < 3:
+            level_names = filter(lambda x: isinstance(x,str), logging._levelNames)
+        else:
+            level_names = list(logging._levelToName.values())
```

Comment:
```suggestion
            level_names = logging._levelToName.values()
```
There is a sorted() below, so no need for list(). Won't this work in py2? :pleading_face: